draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -784,4 +798,73 @@ private void notifyCloseToLeftListener() {
   public int getDraggedViewHeightPlusMarginTop() {
     return transformer.getMinHeightPlusMargin();
   }
+
+  public void initializeTimers(){
+
+    onMaximizeCountdownTimer = new CountDownTimer(2000, 10) {
+      @Override
+      public void onTick(long l) {
+
+        if (getSecondViewAlpha() > 0.95) {
+          notifyMaximizeToListener();
+          new Handler().post(new Runnable() {
+            @Override
+            public void run() {
+              onMaximizeCountdownTimer.cancel();
+            }
+          });
+        }
+      }
+
+      @Override
+      public void onFinish() {
+
+      }
+    };
```

Comment:
@pedrovgs While I'm coding the changes through, I will add a sample for ExoPlayer along with others for the purpose of improving usage of both components together. I believe since industry is focused on using ExoPlayer based players, so this could be great to give it a full support!